src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -175,25 +203,25 @@ public static Text of() {
      * @return The built text object
      */
     public static Text of(Object... objects) {
+        // Shortcut for lonely TextRepresentables
+        if (objects.length == 1 && objects[0] instanceof TextRepresentable) {
```

Comment:
I'm having trouble parsing what you said, but let me try to reply. First off, I have a hard time believing that someone would not recognize that you can call toText on TextRepresentable (literally a single-method interface). In any case, I don't find this to be a good reason to add more functionality and make things more complicated for the sake of redundancy.

The big problem I have with this is that now it is very hard for people to define custom formats for any object marked as TextRepresentable, because it is always passed into Texts.of and converted before they have any chance to do anything about it. So instead they have to convert the object manually before using it with any formatting or Text methods, which defeats the entire purpose of using custom formats in the first place.